frontend/views/containers/payments/MonthOverview.vue

```diff
@@ -0,0 +1,170 @@
+<template lang='pug'>
+.c-summary
+  i18n.c-summary-title.is-title-4(
+    tag='h4'
+    data-test='thisMonth'
+    :args='{ month: thisMonth() }'
+  ) {month} overview
+
+  .c-summary-item(
+    v-for='(item, index) in paymentSummary'
+    :key='index'
+  )
+    h5.label {{ item.title }}
+
+    progress-bar.c-progress(
+      :max='item.max'
+      :value='item.value'
+      :hasMarks='item.hasMarks'
+    )
+    p(:class='{"has-text-success": item.max === item.value}')
+      i.icon-check(v-if='item.max === item.value')
+      .has-text-1 {{ item.label }}
+</template>
+
+<script>
+import currencies from '@view-utils/currencies.js'
+import { mapGetters } from 'vuex'
+import ProgressBar from '@components/graphs/Progress.vue'
+
+export default {
+  name: 'MonthOverview',
+  components: {
+    ProgressBar
+  },
+  data () {
+    return {
+      // Temp
+      monthsNames: ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'],
+      fakeStore: {
+        usersToPay: [
+          {
+            name: 'Lilia Bouvet',
+            avatar: '/assets/images/default-avatar.png',
+            status: 'todo',
+            amount: 10
+          },
+          {
+            name: 'Charlotte Doherty',
+            avatar: '/assets/images/default-avatar.png',
+            status: 'todo',
+            amount: 20
+          },
+          {
+            name: 'Kim Kr',
+            avatar: '/assets/images/default-avatar.png',
+            status: 'rejected',
+            amount: 25
+          },
+          {
+            name: 'Zoe Kim',
+            avatar: '/assets/images/default-avatar.png',
+            status: 'pending',
+            amount: 30
+          },
+          {
+            name: 'Hugo Lil',
+            avatar: '/assets/images/default-avatar.png',
+            status: 'completed',
+            amount: 50
+          }
+        ]
+      }
+    }
+  },
+  methods: {
+    // TEMP
+    thisMonth () {
+      return this.monthsNames[new Date().getMonth()]
+    },
+    statusIsSent (user) {
+      return ['completed', 'pending'].includes(user.status)
+    },
+    statusIsCompleted (user) {
+      return user.status === 'completed'
+    }
+  },
+  computed: {
+    ...mapGetters([
+      'groupSettings'
+    ]),
+    currency () {
+      return currencies[this.groupSettings.mincomeCurrency].symbol
+    },
+    paymentSummary () {
+      const { sent, confirmed, amoutSent, amountTotal, paymentsTotal } = this.paymentStatus
+
+      return [
+        {
+          title: this.L('Payments completed'),
```

Comment:
As a matter of style consistency and readability, could you replace instances of `this.L` with `L`?